src/handlebars/installers.handlebars

```diff
@@ -0,0 +1,169 @@
+        <p>5. Click <strong>Install</strong> to begin the installation.</p>
+        <p>6. When the installation is finished, click <strong>Finish</strong> to close the program.</p>
+
+
+     <h4 id="windows-msi-cmdline">Command-line installation</h4>
+
+        <p>A silent installation allows you to install the Windows package with pre-selected features without user interaction, which can be useful for widescale
+          deployment. Follow these steps:
+
+        <p>1. Download the <tt>.msi</tt> file.</p>
+        <p>2. Choose the features that you want to install, which are shown in the following table:
+            <br>
+              <table>
+
+                <tr>
+                  <th>Feature</th>
+                  <th>Description</th>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureMain</tt></td>
+                  <td>Core OpenJDK installation (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureEnvironment</tt></td>
+                  <td>Update the <strong>PATH</strong> environment variable (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJarFileRunWith</tt></td>
+                  <td>Associate <tt>.jar</tt> files with Java applications (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJavaHome</tt></td>
+                  <td>Update the <strong>JAVA_HOME</strong> environment variable</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureIcedTeaWeb</tt></td>
+                  <td>Install <a href="https://www.github.com/adoptopenjdk/icedtea-web" target="_blank">IcedTea-Web</a></td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJNLPFileRunWith</tt></td>
+                  <td>Associate <tt>.jnlp</tt> files with IcedTea-web</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureOracleJavaSoft</tt></td>
+                  <td>Updates registry keys HKLM\SOFTWARE\JavaSoft\</td>
+                </tr>
+
+              </table>
+        </p>
+
+        <p>Note: <tt>FeatureOracleJavaSoft</tt> can be used to prevent Oracle Java launching from PATH when AdoptOpenJDK is uninstalled. Reinstall Oracle Java if you need
+        to restore the Oracle registry keys.</p>
+
+        <p>Optional parameters can be used that group some of the features together:
+            <br>
+            <table>
+              <tr>
+                <th>Parameter</th>
+                <th>Features</th>
+              </tr>
+
+              <tr>
+                <td>INSTALLLEVEL=1</td>
+                <td>FeatureMain,FeatureEnvironment,FeatureJarFileRunWith</td>
+              </tr>
+
+              <tr>
+                <td>INSTALLLEVEL=2</td>
+                <td>FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome</td>
```

Comment:
FeatureIcedTeaWeb is missing ( I think it will be LEVEL2 .. to check )

Reply:
FeatureIcedTeaWeb  and FeatureJNLPFileRunWith  are actually at INSTALLLEVEL=2
FeatureJNLPFileRunWith (intentionnaly ?)   is still missing from ROW INSTALLLEVEL=2
You can add it at LEVEL2.

I suggest @charliegracie to change FeatureJNLPFileRunWith  to INSTALLLEVEL=3 because it is deeper in the tree
@SueChaplain if @charliegracie will change it we will make a new update for the website

Reply:
Modifications made to add INSTALLLEVEL=3 for FeatureJNLPRunWith. Resolving this conversation.